blocks/product-assortment-carousel-block/features/product-assortment-carousel/default.jsx

```diff
@@ -21,7 +21,7 @@ const BLOCK_CLASS_NAME = "b-product-assortment-carousel";
 const MIN_SLIDES = 4;
 const MAX_SLIDES = 12;
 
-const getProductURL = (item) => item.filter((x) => x.name === "product_url")[0]?.value || {};
+const getProductURL = (item) => item.filter((x) => x.name === "product_url")[0]?.value || "#";
```

Comment:
After adding a test case against this fallback of `{}` surfaced an assumption with the `Link` component that would choke on this object passing in as a fallback.
![Screen Shot 2022-07-27 at 4 48 44 PM](https://user-images.githubusercontent.com/2506649/181374713-6a6e7f84-6e29-46e2-8741-2f336506a16e.png)

There was another fallback elsewhere in this component that used `#` so keeping that consistent here.